src/entity/Quiz/api.ts

```
@@ -2,13 +2,27 @@ import { api } from "@shared/api";
+  const route = `/organizations/${params.organization_id}/quizzes`;
 
-  delete params['organization_id']
+  delete params["organization_id"];
 
   return await api.post<Quiz>(route, params);
 };
+
+export const updateQuiz = async (params: CreateQuiz & { quizId?: string }) => {
+  const route = `/organizations/${params.organization_id}/quizzes/${params.quizId}`;
+
+  delete params["organization_id"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)